components/brave_wallet_ui/common/hooks/send.ts

```diff
@@ -381,6 +381,7 @@ export default function useSend () {
     const amountBN = new Amount(sendAmount)
       .multiplyByDecimals(selectedSendAsset.decimals) // ETH → Wei conversion
       .value // extract BigNumber object wrapped by Amount
+    // @ts-ignore
```

Comment:
What are we ignoring?

Reply:
`amountBN.decimalPlaces()` could be `null` which is not comparable with numbers. I ignored it, since I didn't want to introduce unrelated changes in the PR.